packages/core/src/base-chart.ts

```diff
@@ -363,6 +367,32 @@ export class BaseChart {
 		resizeObserver.observe(this.holder);
 	}
 
+	/**
+	 * Draws title within the Chart's svg element.
+	 * @param title
+	 */
+	drawTitle(title: string) {
```

Comment:
Overall the way this function works isn't too ideal:
1. What if `title` is updated? Seems like we're only adding a title here and not updating
2. A function that deals with titles is updating the `transform` string for the entire graph

Reply:
1. I will move it outisde the initializing of the chart so it runs anytime the setData is called 

2. is there a better way to move the entire chart rather than translations on the inner-wrap? 

I do think it could be nicer to remove the layout spacing code to its own function and have this **only** draw the title.